assets/src/edit-story/components/canvas/multiSelectionMoveable/index.js

```diff
@@ -65,6 +65,16 @@ function MultiSelectionMoveable({ selectedElements }) {
     actions: { pushTransform },
   } = useTransform();
 
+  useEffect(() => {
+    const handleUpdateMoveable = () => {
+      moveable.current?.updateRect();
+    };
```

Comment:
Could just be:

```js
    const handleUpdateMoveable = () => moveable.current?.updateRect();
```

Reply:
And if it wraps in two lines still, the function name can be shortened significantly to just `update` or similar.